spring-tx/src/main/java/org/springframework/transaction/interceptor/DefaultTransactionAttribute.java

```diff
@@ -206,7 +206,7 @@ public void resolveAttributeStrings(@Nullable StringValueResolver resolver) {
 				}
 				catch (RuntimeException ex) {
 					throw new IllegalArgumentException(
-							"Invalid timeoutString value \"" + timeoutString + "\" - cannot parse into int");
+							"Invalid timeoutString value \"" + timeoutString + "\": " + ex);
```

Comment:
let's use `;` as a separator since `:` is already part of `Throwable::toString`
```suggestion
							"Invalid timeoutString value \"" + timeoutString + "\"; " + ex);
```

Reply:
Ok, I will fix these cases together